### PR TITLE
organize with es6 module #3

### PR DESCRIPTION
**Pull request by gasolin**
I change the common JS export to es6 module.

@raylin could you help review this?

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1 +1,4 @@
-{ "presets": ["es2015"] }
+{
+  "presets": ["es2015"],
+  "plugins": ["add-module-exports"]
```
Comment (raylin) [inline .babelrc:3]: if we use http://babeljs.io/docs/learn-es2015/#modules way to import module, could we omit this dependency?

Comment (gasolin) [inline .babelrc:3]: Without the plugin, all `export default xxx` module need to be import with `require('xxx').default`

Comment (raylin) [inline .babelrc:3]: Indeed, unless use `import xxx from 'xxx'` instead of `require`. Do we consider to change to new syntax??

Comment (gasolin) [inline .babelrc:3]: ouch you are right, this is half backed, thanks for remind the `import` part

Comment (gasolin) [inline .babelrc:3]: This is not the urgent work so I'll close the PR and welcome anyone to make a full patch

**Pull request closed without merging**